package.json

```diff
@@ -47,6 +50,7 @@
   },
   "devDependencies": {
     "@craco/craco": "^7.1.0",
+    "babel-preset-react-app": "@babel/plugin-proposal-private-property-in-object",
```

Comment:
Do you recall why this was needed?

Reply:
I don't remember why I need that here. I will remove it and test the app to see if any error happen or not.